### DISCUSSION TITLE
Help with testing

### DISCUSSION THREAD

**Discussion by eliuzhc** [dmitry-s93/MControlCenter#106] [General]
Hi, i'm testing MControlCenter on a SUMMIT E15  A11SCS-402IT, where and how can I help you to test it?  It's a very good project and I want to contribute as much as I can

**Comment by mutchiko** [comment]
If _somehow_ you're still interested, follow the steps to request support for your device in [msi-ec](https://github.com/BeardOverflow/msi-ec)
